.reuse/dep5

```diff
@@ -0,0 +1,90 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: DebOps
+Upstream-Contact: Maciej Delmanowski <drybjed@gmail.com>
+Source: https://github.com/debops/debops
+
+Files: README.md
+Copyright: 2018-2020, Maciej Delmanowski <drybjed@gmail.com>
+           2018-2020, DebOps <https://debops.org/>
+License: GPL-3.0-or-later
+
+Files: ansible/roles/dokuwiki/files/srv/www/dokuwiki/farm/animal/*
+Copyright: 2004-2010 Andreas Gohr <andi@splitbrain.org>
+License: GPL-2.0-only
+
+Files: ansible/roles/owncloud/files/usr/local/src/owncloud/keyring/0x*
+Copyright: 2015-2016 Maciej Delmanowski <drybjed@gmail.com>
+Copyright: 2015      Hartmut Goebel <h.goebel@crazy-compilers.com>
+Copyright: 2015-2019 Robin Schneider <ypid@riseup.net>
+Copyright: 2015-2019 DebOps <https://debops.org/>
+License: GPL-3.0-or-later
```

Comment:
Not sure if GPL is the best license for OpenPGP keys. Maybe state that it is under Public Domain?

Reply:
There is no "Public Domain" license in SPDX list [due to reasons](https://wiki.spdx.org/view/Legal_Team/Decisions/Dealing_with_Public_Domain_within_SPDX_Files). In my opinion these keys should be removed entirely from the repository - the `keyring` role can deal with GPG keys not published on the keyserver network by adding them to the local keyring repository. The role can mention that these GPG keys need to be downloaded separately. But since for now I wanted to capture the existing state which can be changed later, I went with the same license as the role itself.

I'm not sure what license should be applied to GPG keys... What do you think about [CC0](https://creativecommons.org/share-your-work/public-domain/cc0/) instead?

Reply:
CC0 is also fine. Actually, when I said Public Domain, I already included CC0 in my thought :)

Reply:
OK, will do.